modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaPlayFrameworkCodegen.java

```diff
@@ -153,6 +153,9 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("routes.mustache", "conf", "routes"));
 
         //App/Utils folder
+        if (!this.controllerOnly && this.useInterfaces) {
```

Comment:
Unless I'm mistaken, I don't think it's possible. Since we use injection, if I inject the interface, I must be able to tell Play which implementation implements the interface (in the module). If only the interface exists, Play will not be able to run.

For your problem, you just have to use the normal "swagger-codegen-ignore" file that will ignore file you put in it, like a git ignore.